canonical-web-and-design/maas-ui
# feat(ui): focus on the username when displaying the login form

## Done

- Upgrade react-components to 0.9.0.
- Focus on the username field when displaying the login form.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit the UI while logged out.
- You should see the login form and the username field should get focus.

## Fixes

Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/1469.
